engine/src/main/java/com/datatorrent/stram/plan/logical/LogicalPlan.java

```diff
@@ -405,6 +412,7 @@ public void sendMetrics(Collection<String> metricNames)
     private String persistOperatorName;
     public Map<InputPortMeta, OperatorMeta> sinkSpecificPersistOperatorMap;
     public Map<InputPortMeta, InputPortMeta> sinkSpecificPersistInputPortMap;
+    private String moduleName;  // Name of the module which has this stream. null if top level stream.
```

Comment:
This is a meta data of the stream which tells which module does this belong to.
As such its not used. But similar variable is added for OperatorMeta and ModuleMeta. Hence to keep things consistent, its added here too.